tests/integration/009-api-update-loyalty-member-db.spec.ts

```diff
@@ -0,0 +1,194 @@
+				data: { age: 4 },
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(400);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe('Age cant be lower than 5');
+		});
+
+		test('Should return error when updating a member with a negative age', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/5`, {
+				data: { age: -1 },
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(400);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe('Age cant be lower than 5');
+		});
+
+		test('Should return error when updating a member with the same phone number', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/1`, {
+				data: { phoneNumber: '0812345678' },
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(400);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe('PhoneNumber should not be the same');
+		});
+
+		test('Should return error when updating a member with passport data when no passport is registered', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/23`, {
+				data: { passportNumber: 'A12345678', passportExpiryDate: '2030-01-01' },
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(400);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe(
+				'No Passport number or Expiry found, Please register passport first.'
+			);
+		});
+
+		test('Should return error when updating a member with expired passport', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/1`, {
+				data: { passportExpiryDate: '2020-01-01' }, // Expired date
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(400);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe(
+				'Passport Expiry cant be lower or equal to current year.'
+			);
+		});
+
+		test('Should return not found when updating a non-existent member', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/9999`, {
+				data: {
+					firstName: 'UpdatedFirstName',
+					lastName: 'UpdatedLastName',
+					nationality: 'Thai',
+					age: 30,
+					phoneNumber: '0987654321',
+					passportNumber: 'A12345678',
+					passportExpiryDate: '2030-12-15T00:00:00Z',
+				},
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(404);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe('Registered Member is not found.');
+		});
+
+		test('Should return the previous email if the email is not editable', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/1`, {
+				data: { email: 'newemail@gmail.com' },
+			});
+			const body = await res.json();
+
+			await expect(res).toBeOK();
+			expect(body.email).toBe('prinnada.c@gmail.com');
+		});
+
+		test('Should return the previous status is_delete is not editable', async ({
```

Comment:
Please help add more case about 
1) when try to edit the deleted registered member (`is_delete = true`) should be failure. 
2) when try to edit the inactive registered member (`is_active = false`) should be successful.